### PR TITLE
PBM-277 problematic permissions on rpm/deb config files

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: packaging/debian/rules [hunk 1]
```diff
@@ -53,7 +53,7 @@ override_dh_auto_install:
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbmctl $(TMP)/pbmctl
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbm-agent $(TMP)/pbm-agent
 	cp -f build/src/github.com/percona/percona-backup-mongodb/pbm-coordinator $(TMP)/pbm-coordinator
-	cp -f packaging/conf/pbm-agent-storages.conf  $(TMP)/pbm-agent-storages.conf
+	cp -f packaging/conf/pbm-agent-storage.conf  $(TMP)/pbm-agent-storages.conf
```
Comment (tplavcic) [inline packaging/debian/rules]: this:
$(TMP)/pbm-agent-storages.conf
needs to be:
$(TMP)/pbm-agent-storage.conf

File: packaging/rpm/mongodb-backup.spec [hunk 2]
```diff
@@ -206,8 +206,8 @@ esac
 
 %files -n percona-backup-mongodb-agent
 %{_bindir}/pbm-agent
-%config(noreplace) /%{_sysconfdir}/pbm-agent.conf
-%config(noreplace) /%{_sysconfdir}/pbm-agent-storages.conf
+%config(noreplace) %attr(0640,pbm,pbm) /%{_sysconfdir}/pbm-agent.conf
+%config(noreplace) %attr(0640,pbm,pbm) /%{_sysconfdir}/pbm-agent-storages.conf
```
Comment (tplavcic) [inline packaging/rpm/mongodb-backup.spec]: pbm-agent-storages.conf -> pbm-agent-storage.conf

File: packaging/debian/rules (hunk 1 above)
Comment (delgod) [reply packaging/debian/rules]: fixed

File: packaging/rpm/mongodb-backup.spec (hunk 2 above)
Comment (delgod) [reply packaging/rpm/mongodb-backup.spec]: fixed

**Pull request merged**